src/promptflow/tests/sdk_cli_azure_test/e2etests/test_azure_cli_perf.py

```diff
@@ -49,7 +49,7 @@ def operation_scope_args(subscription_id: str, resource_group_name: str, workspa
     "vcr_recording",
 )
 class TestAzureCliPerf:
-    def test_pfazure_run_create(self, operation_scope_args, runtime: str, randstr: Callable[[str], str], time_limit=30):
+    def test_pfazure_run_create(self, operation_scope_args, runtime: str, randstr: Callable[[str], str], time_limit=15):
```

Comment:
how do we determine the `time_limit` and what is it used for? can we add related information in PR description?

Reply:
done